education/cryptocurrency/risk.md

```diff
@@ -14,10 +14,10 @@ Education of how cryptocurrency technology works, how one coin is unique from ot
 - Coin Resources & Skill
 
 ### Price Stability
-While cryptocurrencies differ in supply (limited or unlimited), price stability is largely dependent on supply and demand on trading exchanges as well as an individuals' ability to buy or sell a significant amount of one coin. Since cryptocurrencies differ in design when first launched, investors should consider attributes like coin supply caps and amount of coins the founders are holding (or releasing at a later date) for project advancement purposes. Just like fiat currencies, prices are also impacted by factors outside of the cryptocurrency's control such as major news and announcements, or the coin's dependence on another cryptocurrency.
+While cryptocurrencies vary in supply (limited or unlimited); price stability is based on supply and demand on trading exchanges as well as a person's ability to buy or sell a lot of a specific coin. Since cryptocurrencies differ in design, investors should consider things like coin supply caps and amount of coins the founders are holding (or releasing at a later date) for project advancement purposes. Just like fiat currencies, prices are also impacted by factors outside of the cryptocurrency's control such as major news and announcements, or the coin's dependence on another cryptocurrency.
```

Comment:
This is the paragraph I think needs a rewrite.

There's too many sentence fragments, overall just doesn't flow well for me.